src/utils/clmfates_paraminterfaceMod.F90

```diff
@@ -1,235 +0,0 @@
-module CLMFatesParamInterfaceMod
-  ! NOTE(bja, 2017-01) this code can not go into the main clm-fates
-  ! interface module because of circular dependancies with pftvarcon.
```

Comment:
Wow, you were able to remove this entire file?! That's amazing.

Reply:
So basically this file that's HLM specific gets replaced by FatesReportParameters which is in FATES so it's reused by all of the HLM's, which is a great direction to go.